pkg/scheduler/framework/session.go

```diff
@@ -234,15 +244,23 @@ func (ssn *Session) Pipeline(task *api.TaskInfo, hostname string) error {
 }
 
 //Allocate the task to the node in the session
-func (ssn *Session) Allocate(task *api.TaskInfo, hostname string) error {
+func (ssn *Session) Allocate(task *api.TaskInfo, node *api.NodeInfo) error {
```

Comment:
it is better to have `Backfill` func for that.

Reply:
being discussed elsewhere. resolved to prevent dup.